deploy/crds/scheduling-scale-performance00.crd.yaml

```diff
@@ -14,7 +14,11 @@ spec:
     singular: ssp
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - additionalPrinterColumns:
+    - JSONPath: .status.phase
```

Comment:
hi @borod108 / @akrejcir ,
There is an error in the CRD definition for this property:
`JSONPath` --> `jsonPath`
```
error: error validating "_out/crds/scheduling-scale-performance00.crd.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec.versions[0].additionalPrinterColumns[0]): unknown field "JSONPath" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceColumnDefinition, ValidationError(CustomResourceDefinition.spec.versions[0].additionalPrinterColumns[0]): missing required field "jsonPath" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceColumnDefinition]
```

Reply:
From where did you get this CRD?
The released CRD `olm-crds.yaml` has correct spelling `jsonPath`: https://github.com/kubevirt/ssp-operator/releases/tag/v0.14.0
And also I see correct spelling when searching through the SSP source code.

Reply:
@akrejcir here it is: https://github.com/kubevirt/ssp-operator/blob/6c9da3933bcc4176499955e79e3ab99bdccd381c/api/v1beta1/ssp_types.go#L98

Reply:
I've fixed the csv-generator. Can you try the new release?

Reply:
done, thanks. Let's wait for CI results